### ISSUE TITLE
How to apply it to other downstream tasks:

### ISSUE THREAD

**Issue by asher-bit** [Video-MAC/VideoMAC#2]
Hello, thank you very much for your work. I would like to try applying this network to other downstream tasks. 
Do I need to retrain the network? Could you please provide the pre-trained network model?
During inference, do I only need to use the target encoder and connect it to the corresponding task decoder?
Thank you very much!

**Comment by PGSmall** [comment]
Please refer to [videomac](https://github.com/NUST-Machine-Intelligence-Laboratory/VideoMAC/issues/2).

**Comment by asher-bit** [comment]
Thank you for your response. I apologize as I am a beginner in this area. Can I understand it as you applied MASK to the input during the pre-training phase to obtain a pre-trained model, and during the inference phase, you can directly use this pre-trained model to obtain feature vectors?

**Comment by PGSmall** [comment]
That's right. The inference phase only requires the features of the video frames to be extracted using the VideoMAC model, and then the downstream tasks, such as VOS, can be performed using the label propagation method ([CRW](https://github.com/ajabri/videowalk)).

**Comment by asher-bit** [comment]
I understand, thank you for your response!
